pkg/arvo/sys/hoon.hoon

```diff
@@ -910,27 +915,38 @@
     (sub len (met boz dat))
   (swp boz dat)
 ::
-::  Like `rip` but produces n-bit blocks instead of 2^n bit blocks.
-::
-++  ripn
-  ~/  %ripn
-  |=  [bits=@ud x=@]
-  ^-  (list @)
-  ?:  =(0 x)  ~
-  [(end 0 bits x) $(x (rsh 0 bits x))]
-::
 ++  rip                                                 ::  disassemble
   ~/  %rip
-  |=  [=bloq x=@]
+  |=  [a=bite b=@]
+  =/  [=bloq =step]  ?^(a a [a *step])
```

Comment:
Though this in particular seems *much* less clear than `[a 1]`

Reply:
I'm aiming for `$bite` to be an idiom in these jets -- you pass the `bloq` and `step` or the `step` is defaulted for you (where that always means one). But I'm not opposed to writing that as `[a 1]` ...